modules/rules_java/7.5.0/presubmit.yml

```diff
@@ -0,0 +1,16 @@
+matrix:
+  platform:
+  - centos7
+  - debian10
+  - macos
+  - ubuntu2004
+  - windows
+  bazel:
```

Comment:
We are dropping 6.x intentionally, right?

Reply:
Newer versions of `rules_java` no longer work with Bazel 6. See L4 in modules/rules_java/7.5.0/MODULE.bazel above.

Reply:
Got it, thanks!